curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6145ed1f22caab087630aaad.md

```diff
@@ -17,10 +17,10 @@ You should use the `p::before` selector.
 assert.exists(new __helpers.CSSHelp(document).getStyle('p::before'));
 ```
 
-You should give the `p::before` pseudo-element a `content` property of `Question #`.
+You should give the `p::before` pseudo-element a `content` property of `"Question #"`.
 
 ```js
-assert.include(new __helpers.CSSHelp(document).getStyle('p::before')?.content, 'Question #');
+assert.include(new __helpers.CSSHelp(document).getStyle('p::before')?.content, '"Question #"');
```

Comment:
woops, quick question to the more knowledgeable here - are the "" needed here on this line?

Reply:
Nope, https://developer.mozilla.org/en-US/docs/Web/API/HTMLTemplateElement/content

Reply:
You are right, I don't know why I included quotes in whitespace characters in my head 😅

> The white space is only replaced, when the getPropVal method is used, and only when true is passed as the second argument.

👍 

> Requesting changes to block this until someone can confirm this does not break cases using single quotes etc.